### ISSUE TITLE
Fire webhooks on federation events

### ISSUE THREAD

**Issue by jeyemwey** [owncast/owncast#1600]
While not immediately necessary, I think that this would be helpful for our webhook users (e.g. for monitoring events) to be notified about federation events.

To do this:

- [ ] decide on which federation event should get a webhook
- [ ] decide on a schema that's send to the webhook
- [ ] create a/multiple new webhook event for federation
- [ ] add integration tests for the new webhooks (if required)
- [ ] display new events in admin interface ([code is here](https://github.com/owncast/owncast-admin/blob/develop/pages/webhooks.tsx#L22))
- [ ] document the new behavior on the website ([code is here](https://github.com/owncast/owncast.github.io/blob/master/content/thirdparty/webhooks.md))

**Timeline:** Developing this can only be started once the software design in the [activity-pub branch](https://github.com/owncast/owncast/tree/gek/activity-pub-1) is finalized, and should (IMO) not block the release of `v0.0.11`.

**Comment by gabek** [comment]
Correct, it's not a part of any milestone and nothing depends on it.

**Comment by gabek** [comment]
https://github.com/owncast/owncast/pull/4362 adds a webhook for new follower events.
